test/cluster/test_crash_coordinator_before_streaming.py

```
@@ -51,18 +90,23 @@ async def test_kill_coordinator_during_op(manager: ManagerClient, failure_detect
     coordinator_host = await get_coordinator_host(manager)
     other_nodes = [srv for srv in nodes if srv.server_id != coordinator_host.server_id]
-    num_elections = len(await get_coordinator_host_ids(manager))
-    await manager.api.enable_injection(coordinator_host.ip_addr, "crash_coordinator_before_stream", one_shot=True)
-    await manager.decommission_node(server_id=other_nodes[-1].server_id, expected_error="Decommission failed. See earlier errors")
-    await wait_new_coordinator_elected(manager, num_elections + 1, time.time() + scale_timeout(60))
+    armed = await arm_injection(manager, await manager.running_servers())
+    try:
+        await manager.decommission_node(server_id=other_nodes[-1].server_id, expected_error="Decommission failed. See earlier errors")
+    except ClientConnectionError:
+        # The decommissioned node was the coordinator and crashed on the injection.
+        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(ClientConnectionError)` instead of `try`-`except`-`pass`

(SIM105)